app/src/main/res/values/strings.xml

```diff
@@ -571,39 +571,37 @@
     <string name="please_read">PLEASE READ!</string>
     <string name="fasthub_faq_description">
         <![CDATA[
-        <h5>• Why can\'t I see my <b>Organizations</b> either <i>Private</i> or <i>Public</i> ones?</h5>
-        <p>Open up https://github.com/settings/applications and look for FastHub, open it then scroll to Organization access and click on Grant Button,
-        alternatively login via <b>Access Token</b> which will ease this setup.</p>
+        <h5>• Why can I see neither my <i>Private</i> nor my <i>Public</i> <b>Organizations</b>?</h5>
+        <p>Open up https://github.com/settings/applications and look for FastHub, open it up, scroll down to "Organization access" and click on the "Grant" button.
+        Alternatively you can sign in using <b>Access Token</b> which will ease the process.</p>
 
-        <h5>• I tried to login via Access Token &amp; OTP but it does not work?</h5>
-        <p>You can\'t login via Access Token &amp; OTP all together due to the lifetime of the OTP code, you\'ll be required to login in every few seconds.</p>
+        <h5>• I have tried signing in with an Access Token and a One-Time Password (OTP) without any luck. What's wrong?</h5>
+        <p>You can\'t sign in using Access Token and OTP all together. Due to the limited lifetime of such passwords you\'ll have to repeat a sign in procedure every few seconds.</p>
 
-        <h5>• Why are my Private Repo &amp; Enterprise Wiki not showing up?</h5>
-        <p>It\'s due to FastHub scraping GitHub Wiki page &amp; Private Repos require session token that FastHub is unable to obtain.</p>
+        <h5>• Why isn\'t my Private/Enterprise Repository\'s Wiki showing up?</h5>
```

Comment:
I still think `aren't` is the correct way to say it

Reply:
There's only a single Wiki in the repo. Usage of `aren't` is contextually incorrect.

Reply:
Nope disagree. You are talking about **both** Private AND Enterprise.

Reply:
A privare repo OR an enterprise repo.  
Here's how the sentences look like without OR:
`Why isn\'t my Private Repository\'s Wiki showing up?` AND `Why isn\'t my Enterprise Repository\'s Wiki showing up?`

Reply:
That would be the correct way to use isn't. since you are putting both in one is uses `aren't`

Reply:
Okey, another example.  
Let's remove the adjectives which describe a repository's state.  
`Why isn\'t my Repository\'s Wiki showing up?`  
See? There's only a single repo, not two.

Reply:
that would be correct

Reply:
Let's add them back again.  
`Why isn\'t my Private/Enterprise Repository\'s Wiki showing up?`  
A single repo remains.

Reply:
No :) now you are saying 2 things LOL....
isn't would be correct if it would be written like this 

Why isn\'t my Private or my Enterprise

Reply:
`/` reduces `or my` in this sentence.

Reply:
/ = `or` not `or my` LOL what ever....

Reply:
I'm going to edit this sentence into
`Why aren\'t my Private and Enterprise Repositories\' Wikis showing up?`

Reply:
I was wrong. `/` reduces `or`, not `or my`. But still we were talking about one repo's wiki, so my variant was absolutely correct.